jujuclient/all.go

```diff
@@ -18,3 +36,51 @@ type store struct {
 var DefaultControllerStore = func() (ControllerStore, error) {
 	return &store{}, nil
 }
+
+func acquireEnvironmentLock(lockName, operation string) (*fslock.Lock, error) {
```

Comment:
why environment? because copy and paste? if so -- please be intentional, and use meaningful names